changelogs/fragments/597_remove_fstab_comment_on_updating.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - fstab - If a comment is appended to a fstab entry, state present creates a double-entry (https://github.com/ansible-collections/ansible.posix/issues/595).
```

Comment:
- mount - If a comment is appended to a fstab entry, state present creates a double-entry (https://github.com/ansible-collections/ansible.posix/issues/595).

I think it would be better to replace `fstab` with `mount` as the affected module name.

Reply:
You're right. I fixed the fragment. Sorry for the noise. Thank you!